Makefile

```diff
@@ -43,6 +44,7 @@ NODE_EXE = node$(EXEEXT)
 NODE ?= ./$(NODE_EXE)
 NODE_G_EXE = node_g$(EXEEXT)
 NPM ?= ./deps/npm/bin/npm-cli.js
+TEST_CI = ./tools/test-ci.sh
```

Comment:
> If you want honest advice: Close this PR

Yeah I've already put it on the back burner https://github.com/nodejs/node/pull/12874#issuecomment-299665089

Just kicking around the idea...